crossbot/models.py

```
@@ -453,3 +509,91 @@ def __str__(self):
 
         return '*{}* by {}{}:\n {}'.format(self.name, self.user, arg_str,
                                            self.command)
+
+
+# Items are stored in YAML (not the DB) but loaded here for convenience
+
+
+class Item:
+    ITEMS = {}
+
+    def __init__(self, key, options):
+        """Only used on initialization, do not call elsewhere."""
+        self.key = key
+        self.name = options['name']
+        # use setattr and defaults in getter methods?
+        self.droppable = options.get('droppable', True)
+        self.image_name = options.get('image_name', None)
+        self.rarity = options.get('rarity', 1.0)
+        self.type = options.get('type', None)
+
+    @classmethod
+    def load_items(cls):
+        with open(path.join(path.dirname(__file__), 'items.yaml')) as f:
+            for key, options in yaml.load(f).items():
+                cls.ITEMS[key] = Item(key, options)
+
+    @classmethod
+    def from_key(cls, key):
+        return cls.ITEMS.get(key, None)
```

This should raise `KeyError` instead of return None